origin_test.go

```
@@ -111,32 +122,31 @@ func TestRepositoryOrigin(t *testing.T) {
-		want   *PackageOrigin
+		origin *DependencyOrigin
+		want   *DependencyOrigin
 	}{
 		{name: "nil"},
-		{name: "empty", origin: &PackageOrigin{}},
-		{name: "credentialed artifact", origin: &PackageOrigin{ArtifactURL: "https://build:s3cret@nexus.corp/pkg.tgz"}},
-		{name: "local repository", origin: &PackageOrigin{Repository: "file:///home/someone/repo"}},
-		{name: "revision without a repository", origin: &PackageOrigin{Revision: "9f8e7d6"}},
-		{name: "disputed reports nothing", origin: &PackageOrigin{Disputed: true, ArtifactURL: "https://registry.npmjs.org/react/-/react-18.2.0.tgz"}},
+		{name: "empty", origin: &DependencyOrigin{}},
+		{name: "credentialed artifact", origin: &DependencyOrigin{ArtifactURL: "https://build:s3cret@nexus.corp/pkg.tgz"}},
```

golangci-lint (2.12.2)

[high] G101: Potential hardcoded credentials: Password in URL

(gosec)